core/scripts/creator-umip/1_Propose.js

```diff
@@ -0,0 +1,59 @@
+// This script generates and submits an identifier-add upgrade transaction to the DVM. It can be run on a local ganache
+// fork of the main net or can be run directly on the main net to execute the upgrade transactions.
+// To run this on the localhost first fork main net into Ganache with the proposerWallet unlocked as follows:
+// ganache-cli --fork https://mainnet.infura.io/v3/d70106f59aef456c9e5bfbb0c2cc7164 --unlock 0x2bAaA41d155ad8a4126184950B31F50A1513cE25
+// Then execute the script as: truffle exec ./scripts/creator-umip/1_Propose.js --network mainnet-fork --creator 0x0139d00c416e9F40465a95481F4E36422a0A5fcc from core
+
+const Registry = artifacts.require("Registry");
+const Governor = artifacts.require("Governor");
+
+const argv = require("minimist")(process.argv.slice(), { string: ["creator"] });
```

Comment:
So this `creator` will be the new EMPCreator contract address?

Reply:
Yep, exactly.